common/fs/src/tail.rs

```diff
@@ -139,13 +139,13 @@ impl Tailer {
                                             Lookback::SmallFiles => {
                                                 let offset = match initial_offsets.as_ref() {
                                                     Some(initial_offsets) => {
-                                                        let offset = initial_offsets.get(&path.as_os_str().as_bytes().into()).copied().unwrap_or(0);
-                                                        debug!("Got offset {} from state for {:?} using key {:?}", offset, name, path);
+                                                        let offset = initial_offsets.get(&inode).copied().unwrap_or(0);
+                                                        debug!("Got offset {} from state for {:?} using key {:?} for path {:?}", offset, name, inode, path);
                                                         offset
                                                     }
                                                     None => {
                                                         let len = path.metadata().map(|m| m.len()).unwrap_or(0);
-                                                        debug!("Smallfiles lookback {} from state for {:?} using key {:?}", len, name, path);
+                                                        debug!("Smallfiles lookback {} from len for {:?} using key {:?}", len, name, path);
```

Comment:
oh, this message got me a few times while testing, thanks for taking care of it.